### PR TITLE
Feature: Get Thread likers

### PR DESCRIPTION
**Pull request by Corard**

### CODE REVIEW

File: threads-api/__test__/getThreadLikers.test.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+    const postID = '3141675920411513399';
+
+    // when
+    const likers = await threadsAPI.getThreadLikers(postID);
+
+    // then
+    expect(Array.isArray(likers.users)).toBe(true);
+    expect(likers.users[0]).toHaveProperty('pk');
+    expect(likers.users[0]).toHaveProperty('full_name');
+})
```
Comment (junhoyeo) [inline threads-api/__test__/getThreadLikers.test.ts:15]: Can you run `yarn format`?

File: threads-api/src/threads-api.ts [hunk 2]
```diff
@@ -236,4 +245,26 @@ export class ThreadsAPI {
...
+      'https://www.threads.net/api/graphql',
+      new URLSearchParams({
+        lsd: this.fbLSDToken,
+        variables: `{"mediaID":"${postID}"}`,
+        doc_id: '9360915773983802',
+      }),
+      {
+        headers: {
+          ...this._getDefaultHeaders(),
+          'x-fb-friendly-name': 'BarcelonaPostPageQuery',
```
Comment (junhoyeo) [inline threads-api/src/threads-api.ts:263]:
```suggestion
```
nit, but I think the query name is wrong. Please remove this if you are not familiar with it!

File: threads-api/__test__/getThreadLikers.test.ts (hunk 1 above)
Comment (Corard) [reply threads-api/__test__/getThreadLikers.test.ts]: Just done that now :)

**Review by junhoyeo — APPROVED** [review]
LGTM! I'll be glad to merge this PR if you address some small feedbacks.

**Review by junhoyeo — APPROVED** [review]
@Corard Thank you so much!

**Comment by junhoyeo** [comment]
@Corard Just deployed [v1.0.8](https://github.com/junhoyeo/threads-api/releases/tag/v1.0.8) with it!

**Pull request merged**